pages/sight.vue

```diff
@@ -263,52 +257,6 @@
           </div>
         </v-col>
       </v-row>
-
-      <v-row>
-        <v-col cols="12" sm="12">
-          <v-card-text class="text-center" style="margin: 3rem auto"
-            ><h1 style="text-decoration: underline">
-              Get Involved
```

Comment:
Please bring back this section. This was not supposed to be removed.

Reply:
And I was saying about "Get Involved" type of heading.
You need not to insert headings in the card.

Reply:
> Please bring back this section. This was not supposed to be removed
Actually, I was supposed to remove these three columns and merge it into a single  "Get Involved" column. But the matter provided is  barely of two lines and it's looking very odd.